test/testpy/__init__.py

```diff
@@ -53,11 +53,15 @@ def AfterRun(self, result):
     # delete the whole tmp dir
     try:
       rmtree(self.tmpdir)
+      if "NODE_PIPE_DIR" in os.environ:
+        rmtree(os.environ["NODE_PIPE_DIR"]);
```

Comment:
@mdawsonibm Yes, I think any of the two solutions you mentioned would work, with a preference for the sub-directory since it would probably require less maintenance over time (no need to track which files need to be removed if the list of files created changes).